bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/FirstReconciler.java

```diff
@@ -5,7 +5,7 @@
 import io.javaoperatorsdk.operator.api.reconciler.UpdateControl;
 import io.quarkiverse.operatorsdk.annotations.CSVMetadata;
 
-@CSVMetadata(name = "first-operator", version = FirstReconciler.VERSION)
+@CSVMetadata(name = "first-operator", version = FirstReconciler.VERSION, installModes = @CSVMetadata.InstallMode(type = "MultiNamespace", supported = false))
```

Comment:
Maybe there should be a public wrapper around the allowed type values.

Reply:
Do you mean like an enum or something like that?